appveyor.yml

```diff
@@ -5,14 +5,20 @@ branches:
 image: Visual Studio 2017
 configuration: Release
 clone_folder: c:\projects\openscap
+
+environment:
+  matrix:
+    - PYTHON_VERSION: 36
+
 install:
+- set PATH=C:\Python%PYTHON_VERSION%;%PYTHON%\Scripts;%PATH%
 - cmd: vcpkg install curl libxml2 libxslt bzip2 pcre pthreads
```

Comment:
also needs swig I think

Reply:
Doesn't fail on build (kinda expect it to, but anyway). Is it Python that needs SWIG?